.vscode/c_cpp_properties.json

```diff
@@ -4,7 +4,9 @@
             "name": "Linux",
             "includePath": [
                 "${workspaceFolder}/**",
-                "/opt/ros/foxy/include"
+                "/opt/ros/foxy/include",
+                "/home/wsl/mambaforge/envs/driverless_env/include/",
```

Comment:
not a fan of this... Plus we can drop the `/opt/ros/foxy/include` now that we wont be developing in a container. I'd say just gitignore this file and put a thing in the wiki about how to create this file for your specific dev environment (since these paths will be different for everyone)